docs/root/configuration/network_filters/mysql_proxy_filter.rst

```diff
@@ -0,0 +1,31 @@
+.. _config_network_filters_mysql_proxy:
+
+Mysql proxy
+===========
+
+The Mysql proxy filter decodes the wire protocol between the Mysql client
+and server. It does not decode the SQL queries in the payload, as of today.
+
+.. _config_network_filters_mysql_proxy_stats:
+
+Statistics
+----------
+
+Every configured Mysql proxy filter has statistics rooted at *mysql.<stat_prefix>.* with the
+following statistics:
+
+.. csv-table::
+  :header: Name, Type, Description
+  :widths: 1, 1, 2
+
+  new_sessions, Counter, Number of Mysql sessions since start
+  total_mysql_headers, Counter, Number of times the delay is injected
+  byte_count, Counter, Number of OP_GET_MORE messages
```

Comment:
I think maybe you copied some of this from Mongo? Can you fix the stats docs?

Reply:
ugh yeah.. cleaning these